better-politics-mod/localization/english/BPM_union_leadership_l_english.txt

```diff
@@ -0,0 +1,11 @@
+l_english:
+ bpm_union_leadership.1.t:0 "The Subjugation of Labor"
+ bpm_union_leadership.1.desc:0 "The independent trade unions in our country have been defeated and consolidated into a new appendage of the state. (Write something Better Later)"
+ bpm_union_leadership.1.f:0 "Insert a quote from Robert Ley or one of the Strasser Brothers here.\n\n—Limabot, 2024"
+ bpm_union_leadership.1.a:0 "Good for us."
+
+
+bpm_union_leadership.2.t:0 "The Liberation of Labor"
```

Comment:
As mentioned in dev chat, this should probably be different for illegal unions, socialist unions, and syndicalism. See dev chat for triggered desc details